source/components/Login/_tests/Login.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import sinon from 'sinon'
+import { expect } from '../../../../configuration/testSetup'
+import LoginContainer from '../LoginContainer'
+
+describe.only( '<LoginContainer />', () => {
+
+  const spy = sinon.spy(LoginContainer, 'redirect')
+
+  const wrapper = mount( <LoginContainer /> )
+
+  it( 'should render a <Heading />', () =>
+    expect( wrapper.find( 'Heading' ).length ).to.equal( 1 )
+  )
+
+  it( 'should render login component with class \'login-container\'', () =>
+    expect( wrapper.find( '.login-container' ).length ).to.equal( 1 )
+  )
+
+  it( 'should render button with class \'login-button\'', () =>
+    expect( wrapper.find( '.login-button' ).length ).to.equal( 1 )
+  )
+
+  it( 'should go to route /auth/google when button clicked', () => {
```

Comment:
We discovered that there was a mistake in the react routing.  We believe this test is still valid.